### PR TITLE
Add datasource to get jboss versions

### PR DESCRIPTION
**Pull request by chenlizhong**
Signed-off-by: Chen lizhong <lichen@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

File: insights/specs/datasources/ps.py [hunk 1]
```diff
@@ -76,3 +78,62 @@ def ps_eo_cmd(broker):
         return DatasourceProvider('\n'.join(data), relative_path='insights_commands/ps_eo_cmd')
 
     raise SkipComponent()
+
+
+@datasource(LocalSpecs.ps_eo_args, HostContext)
+def jboss_runtime_versions(broker):
```
Comment (ryan-blakley) [inline insights/specs/datasources/ps.py:84]: I don't see a parser for this datasource, is this just an analytics spec/datasource?

**Review by ryan-blakley — CHANGES_REQUESTED** [review]

File: insights/specs/datasources/ps.py (hunk 1 above)
Comment (chenlizhong) [reply insights/specs/datasources/ps.py:85]: This is for subscription watch service, I am not sure if need parser. Another thing, I want to collect data, then decide for parser writing.

**Review by xiangce — COMMENTED** [review]
And for new spec (or datasource), it should get the approval in the core-assets repo before adding to the core collection.

File: insights/specs/datasources/ps.py (hunk 1 above)
Comment (xiangce) [reply insights/specs/datasources/ps.py:85]: @chenlizhong - please open an issue for this spec to the core-asserts repo to get it approved.

**Review by xiangce — APPROVED** [review]

**Review by psachin — APPROVED** [review]

**Review by ryan-blakley — APPROVED** [review]

**Pull request merged**